azure-storage-queue/src/main/scala/akka/stream/alpakka/azure/storagequeue/AzureQueueSinkFunctions.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.azure.storagequeue
+
+import com.microsoft.azure.storage.queue.{CloudQueue, CloudQueueMessage}
+import scala.concurrent.{ExecutionContext, Future}
+import akka.Done
+
+sealed trait DeleteOrUpdateMessage
+case object Delete extends DeleteOrUpdateMessage
+case class UpdateVisibility(timeout: Int) extends DeleteOrUpdateMessage
+
+private[storagequeue] object AzureQueueSinkFunctions {
```

Comment:
All these operations are blocking on network and must not be run on the default dispatcher but on a dedicated dispatcher for blocking operations, just like the future inside the `AzureQueueSourceStage`. 

I think the easiest way would be to actually get this right would be to get rid of that `FlowMapECStage`, take an implicit `ActorSystem` to the individual methods, look up the `akka.stream.default-blocking-io-dispatcher` from it and pass that as EC to the `mapAsync`s.

Another option would be to rather wrap these in a custom minimal stage and configure that to run on the blocking dispatcher using attributes (see other existing connectors for samples of this)